### ISSUE TITLE
Been a valued customer

### ISSUE THREAD

**Issue by Rkdheishins** [Masuzu/GBFCombatSniffer#10]
i think i was one of your day 1 customer and had multiple products from you, zooey every month and sarasa every time there was a magnafes, i apologies this probably isnt the place for this tho, but im unsure what happened

i play and use the bots daily with at lease 2 capchas a day for god i dont know how many months but since the new anubus raid came out i couldent get in so i decided to try the new combat sniffer program in hopes of getting in faster

this is probably a error on my end but for countless nights id let it play while i sleep but i have the speakers on so when capcha comes on i hear the notification and i turn it off and on again, again, ive been doing this for months. but last night i used combat sniffer and gave up on it because i couldn't quite get it to work so i gave up using it to get into raids but tried to mvp now.

long story short i turned it off and just used sarasa for the rest of the night, to my surprise i woke up with no capcha and this screen

![ban 080819](https://user-images.githubusercontent.com/28521755/62718095-0aeef980-b9ba-11e9-8f36-90aaf11d874f.PNG)

again something i probably somehow did my self but all i know is ive been doing this for moths and the only thing i did different is try a new product

**Comment by Masuzu** [comment]
All the best :/
